src/main.cpp

```diff
@@ -25,59 +25,62 @@
 
 #include "vcf/file_structure.hpp"
 #include "vcf/validator.hpp"
+#include "vcf/report_writer.hpp"
 
+// TODO any reason why this namespace is indented 4 spaces?
 namespace
 {
     size_t const default_line_buffer_size = 64 * 1024;
     namespace po = boost::program_options;
-    
+
     enum class ValidationLevel { error, warning, stop };
-    
-    
+
+
     po::options_description build_command_line_options()
     {
         po::options_description description("Usage: vcf-validator [OPTIONS] [< input_file]\nAllowed options");
-        
+
         description.add_options()
             ("help,h", "Display this help")
             ("input,i", po::value<std::string>()->default_value("stdin"), "Path to the input VCF file, or stdin")
             ("level,l", po::value<std::string>()->default_value("warning"), "Validation level (error, warning, stop)")
             ("version,v", po::value<std::string>(), "VCF fileformat version to validate the file against (v4.1, v4.2, v4.3)")
+            ("output,o", po::value<std::string>()->default_value("stdout"), "Comma separated values for type of output (sqlite, stdout, silent)")
```

Comment:
Not sure most of the users will understand what "sqlite" means. Maybe we could use "database"?